bokehjs/src/lib/models/tools/gestures/tap_tool.ts

```diff
@@ -43,7 +43,7 @@ export class TapToolView extends SelectToolView {
         if (did_hit && callback != null) {
           const x = r_views[0].coordinates.x_scale.invert(geometry.sx)
           const y = r_views[0].coordinates.y_scale.invert(geometry.sy)
-          const data = {geometries: {...geometry, x, y}, source: sm.source}
+          const data = {geometries: {...geometry, x, y}, source: sm.source, tap_type: this.model.behavior}
```

Comment:
why don't we keep consistent naming, and rename `tap_type` to `behavior` here and below

Reply:
Thank you for the feedback! I will make that change.